src/main/java/com/stefanovskyi/gitdeceiver/GitDeceiver.java

```diff
@@ -13,11 +14,9 @@ public static void main(String args[]) {
 
         fakeRepository.cloneRepository(gitAuthData.getRepositoryName());
         fakeRepository.addRemote(gitAuthData.getRepositoryName());
-        LocalDateTime commitDate = Util.getStartDate(args[5]);
+        LocalDateTime commitDate = Util.getStartDate(args[6]);
```

Comment:
Why did you change parameters ordered number? It will broke workflow for all user that uses previous variant. Please change it back.

Reply:
Done